<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icon/atmtv-h0nup-001.ico</normaloff>:/icon/atmtv-h0nup-001.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>391</width>
     <height>221</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/image/title.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lb_start">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>351</y>
     <width>104</width>
     <height>58</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/image/start.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lb_score">
   <property name="geometry">
    <rect>
     <x>287</x>
     <y>351</y>
     <width>104</width>
     <height>58</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/image/rank.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lb_over">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>341</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/image/gameover.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="lb_choose">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/image/choose.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lb_enter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>700</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/image/welcome.png</pixmap>
   </property>
  </widget>
  <zorder>lb_name</zorder>
  <zorder>lb_start</zorder>
  <zorder>lb_over</zorder>
  <zorder>lb_score</zorder>
  <zorder>lb_choose</zorder>
  <zorder>lb_enter</zorder>
  <zorder>lineEdit</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
